server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -747,13 +747,14 @@ private void mergeAndPush()
     for (Map.Entry<Long, Sink> entry : sinks.entrySet()) {
       final Long intervalStart = entry.getKey();
       if (intervalStart < minTimestamp) {
-        log.info("Adding entry[%s] for merge and push.", entry);
+        log.info("Adding entry [%s] for merge and push.", entry);
         sinksToPush.add(entry);
       } else {
         log.warn(
-            "[%s] < [%s] Skipping persist and merge.",
+            "[%s] >= [%s] Skipping persist and merge for entry [%s]",
```

Comment:
I can change that. I've updated the log msg in latest commit, does it make sense ? or some modification is required ? Thanks.